File: beginner/__1000__/1181/1181.lua
local line = tonumber(io.read()) + 1 -- +1 para acertar a indexação-0
local operation = io.read() -- S para soma e M para media
local grid = {} -- grid 12x13

for r=1, 12 do
    local row = {}
    for c=1, 12 do
        row[c] = tonumber(io.read())
    end
    grid[r] = row
end

local sum = 0
for i=1, 12 do
    sum = sum + grid[line][i]
end

if operation == "S" then print(sum) end
if operation == "M" then print(sum / 12) end 

File: beginner/__1000__/1181/1181.py
line=int(input()) 
operation=input()

COLUMNS=12
current_line=0
count=0

soma = 0.0

while True:
  try:
    current_number = float(input())
    if current_line != line:
      count = count + 1
      if count == COLUMNS:
        current_line = current_line + 1
        count = 0
        
    else:
      soma += current_number
      count = count + 1
      if count == COLUMNS:
        break
      
  except EOFError:
    break

if operation == "S":
  print(f'{soma:.1f}')
else:
  print(f'{soma/12:.1f}')